community_server/src/Model/Entity/StateBalance.php

```diff
@@ -70,7 +71,7 @@ protected function _getDecay()
     
     protected function _getAmountFloat()
     {
-        return floatval($this->amount) / 10000.0;
+        return Number::format(floatval($this->amount) / 10000.0, ['precision' => 2]);
```

Comment:
I assume this uses `round()` which rounds up in cases of `x.xx5 - x.xx9` - this is incorrect since the User would be displayed more GDD then he has.

Reply:
I now use a own function with floor like you have wished for